source/api/cypress-api/selector-playground-api.md

```diff
@@ -67,13 +71,27 @@ Cypress.SelectorPlayground.defaults({
 
 ## Get Selector
 
-Returns you the selector value for a given element.
+Returns you the selector value for a given element as determined by the selector strategy.
+
+For example, consider this HTML fragment.
+
+```html
+<button id='bingo' class='number3'>Cup of tea</button>
+```
+
+With the default selector strategy, the selector value will be `'#bingo'` because IDs have priority over classes.
 
 ```js
-// query for a button
 const $el = Cypress.$('button')
+const selector = Cypress.SelectorPlayground.getSelector($el) // '#bingo'
+```
 
-// do something like console.log()
-const selector = Cypress.SelectorPlayground.getSelector($el)
+With a custom selector strategy that favours classes, the selector value will be `'.number3'`.
 
+```js
+Cypress.SelectorPlayground.defaults({
+  selectorPriority: ['class', 'id']
+})
+const $el = Cypress.$('button')
+const selector = Cypress.SelectorPlayground.getSelector($el) // '.number3'
```

Comment:
I fleshed this out a little with some examples to make it clearer _to me at least_.